src/isaw.facultycv/isaw/facultycv/content/profile.py

```diff
@@ -26,96 +26,101 @@
     ),
 
     atapi.ReferenceField(
-        name = 'ProfileRef',
+        name='ProfileRef',
```

Comment:
Maybe keep whitespace fixes on their own branch (or at least dedicated commits) in the future.